cloudprem/variables.tf

```diff
@@ -233,4 +254,30 @@ variable "environment" {
     condition     = length(var.environment) <= 5
     error_message = "The length of the Environment must be less than 6 characters."
   }
+}
+variable "stack_type" {
+  description = "Specify whether this is a production stack or a development stack. Production stacks have data protection settings enabled but will leave resources behind after a terraform delete operation."
+  type = string
+  default = "production"
+
+  validation {
+    condition = var.stack_type == "dev" || var.stack_type == "production"
+    error_message = "Stack type should be one of: [dev, production]."
+  }
+}
+variable "frontegg_api_key" {
+  description = "If webhooks are enabled, this is the api key for access to Frontegg's servers."
+  default = ""
+  type = string
+}
+variable "frontegg_client_id" {
+  description = "If webhooks are enabled, this is the client_id for access to Frontegg's servers."
+  default = ""
+  type = string
+}
+
+variable "replicated_app_sequence_number" {
+  description = "For fresh installs you can target a specific Replicated sequence for first install. This will not be respected for existing installations"
+  default = "latest"
+  type = string
```

Comment:
Could we get away with an `int` here, and `toString` it where it's used? The sequence numbers are always int valued, right?